src/components/CreateCoursePage/CreateCoursePage.scss

```diff
@@ -0,0 +1,25 @@
+@import "~bootstrap/scss/_functions.scss";
+@import "~bootstrap/scss/_variables.scss";
+
+.create-course-form {
+  label {
+    font-weight: 600;
+  }
+
+  .required {
+    color: $danger;
+    margin-left: 4px;
+  }
```

Comment:
Maybe the following?  Makes elements more themable, in case a theme wants to make text bold in danger case or whatever. Could also do this directly in React, but looks like we use this a few places.

```
@extend .ml-1 // (or 2? not sure right size)
@extend .text-danger
```